application/templates/dashboards/index.html

```diff
@@ -2,27 +2,40 @@
 
 {% block body %}
 
-<h1>Administer dashboards</h1>
+<h1>Your dashboards</h1>
 
-<a href="{{ url_for('dashboard_form') }}" class="btn btn-primary" role="button">Create a dashboard</a>
+<p>
+    <a href="{{ url_for('dashboard_form') }}" class="btn btn-primary" role="button">Create a dashboard</a>
+</p>
 
-{% if dashboards %}
-  <h2>Dashboards</h2>
+<p class="end-section">Manage the data and content to be displayed on your dashboards</p>
 
-  <ul>
-    {% for dashboard in dashboards %}
+{% if dashboards %}
+  <ul id="dashboard-list">
+  {% for dashboard in dashboards %}
       <li>
-        {{ dashboard.title }}
-        <ul class='links'>
-          <li>
-            <a class='inline' href="{{ url_for('dashboard_form', uuid=dashboard.id) }}">Edit</a>
-          </li>
-          <li>
-            <a class='inline' href="{{ url_for('dashboard_clone', uuid=dashboard.id) }}">Clone</a>
-          </li>
-        </ul>
+          <div class="row">
+              <div class="col-sm-12 col-md-7 title">
+                  <strong>
+                      {{ dashboard.title }}
+                  </strong>
+                  <span class="hint">
+                      {% if dashboard.published %}
+                          Published
+                      {% else %}
+                          Unpublished
+                      {% endif %}
+                  </span>
+              </div>
+              <ul class="col-sm-12 col-md-5 buttons">
+                  <li><a class='btn btn-success' href="{{ url_for('dashboard_hub', uuid=dashboard.id) }}">Edit dashboard</a></li>
+                  <li><a class='btn btn-success' href="{{ url_for('dashboard_form', uuid=dashboard.id) }}">Admin edit</a></li>
```

Comment:
What does 'Admin edit' mean?

As a rule, I like to [name interactive instructions in a particular way](http://www.uxbooth.com/articles/the-grammar-of-interactivity/).

Do you think it would help to name that link along those guidelines?